src/terms/hubbard.jl

```diff
@@ -0,0 +1,178 @@
+using LinearAlgebra
+using Random
+
+"""
+Structure for manifold choice and projectors extraction.
+
+Overview of fields:
+- `iatom`: Atom position in the atoms array.
+- `species`: Chemical Element as in ElementPsp.
+- `label`: Orbital name, e.g.: "3S".
+
+All fields are optional, only the given ones will be used for selection.
+Can be called with an orbital NamedTuple and returns a boolean
+  stating whether the orbital belongs to the manifold.
+"""
+@kwdef struct OrbitalManifold
```

Comment:
Not a fan of the term "manifold" because of connections to direct minimization in particular. Is this the standard term in the literature?

Reply:
(also it's just not a manifold in the mathematical sense)